### PR TITLE
enhance:use sonic to make serialization and deserialization

### PR DESCRIPTION
**Pull request by Feizekai**

### CODE REVIEW

**Comment by li-jin-gou** [comment]
ci need to solve

**Comment by Feizekai** [comment]
> ci need to solve

sloved

File: etcd/registry.go [hunk 1]
```diff
@@ -16,13 +16,12 @@ package etcd
...
 	"context"
-	"encoding/json"
-	"sync"
-	"time"
-
+	"github.com/bytedance/sonic"
 	"github.com/cloudwego/hertz/pkg/app/server/registry"
 	"github.com/cloudwego/hertz/pkg/common/hlog"
 	clientv3 "go.etcd.io/etcd/client/v3"
+	"sync"
```
Comment (a631807682) [inline etcd/registry.go:23]: Please format the code
> gofumpt -l -extra -w .

Comment (Feizekai) [reply etcd/registry.go:23]: Do I need format the code in local IDE then commit again ？

Comment (a631807682) [reply etcd/registry.go:23]: yes, format the code locally by the command and push the commit .

Comment (Feizekai) [reply etcd/registry.go]: done it

File: etcd/go.mod [hunk 2]
```diff
@@ -3,6 +3,7 @@ module github.com/hertz-contrib/registry/etcd
 go 1.16
 
 require (
+	github.com/bytedance/sonic v1.8.3 // indirect
```
Comment (a631807682) [inline etcd/go.mod:6]: This doesn't look like an indirect dependency.
> go mod tidy

The version here is not the same as `eureka`? can we both use **v1.8.3**?

File: redis/common.go [hunk 3]
```diff
@@ -17,8 +17,8 @@ package redis
 import (
 	"context"
 	"crypto/tls"
-	"encoding/json"
 	"fmt"
+	"github.com/bytedance/sonic"
```
Comment (welkeyever) [inline redis/common.go:21]: import format

Comment (Feizekai) [reply redis/common.go]: done it

**Review by li-jin-gou — APPROVED** [review]

**Review by a631807682 — APPROVED** [review]

**Review by FGYFFFF — APPROVED** [review]

**Pull request merged**